themes/default/build/sass/forms/_silverstripe-forms.scss

```diff
@@ -1,8 +1,12 @@
-/**
- * For Silverstripe specific styling (to get forms to work as intended)
- * Note: This file should not be used as an example of how to structure css - its
- * just a work around for forms in silverstripe. Remove if/what you can.
- */
+/*
```

Comment:
This should all be the single line format, as we don't want to keep them in compiled css. Though our uglify likely strips it all out anyway...